papis/commands/explore.py

```diff
@@ -25,7 +25,7 @@
     papis explore crossref -h
 
 will tell you which commands are available. Let us suppose that you want to
-look for some documents on Crossref, say some papers of Schroedinger's, and
+look for some documents on Crossref, say some papers of Schrodinger's, and
```

Comment:
I'm not claiming either is correct, but the actual code examples write it as `schrodinger` so figured this is best.

Reply:
The German version should have an umlaut, i.e. `Schrödinger`, but I think `Schroedinger` is the ASCII version (at least I've seen other names with an ö written like that instead?).

Wikipedia doesn't seem to feel strongly about it, so consistency is best here.